LiveSplit.AutoSplitters.xml

```diff
@@ -21913,13 +21913,13 @@
     </AutoSplitter>
     <AutoSplitter>
         <Games>
-            <Game>Awesome Tanks</Game>
+            <Game>Marble Mania</Game>
```

Comment:
Why did you remove this other autosplitter?

Reply:
because i copy and pas te